src/OceanSeaIceModels/CrossRealmFluxes/similarity_theory_turbulent_fluxes.jl

```diff
@@ -151,7 +151,7 @@ end
 
 # The complete bulk coefficient should include also `ψ(ℓ / L)`, but the 
 # JRA55 atmosphere is adjusted to formulae without this last term so we exclude it
-@inline simplified_bulk_coefficient(ψ, h, ℓ, L) = log(h / ℓ) - ψ(h / L) # + ψ(ℓ / L)
+@inline simplified_bulk_coefficients(ψ, h, ℓ, L) = log(h / ℓ) - ψ(h / L) # + ψ(ℓ / L)
```

Comment:
Not sure I agree with this logic. JRA55 is also nudged strongly to observations. I think we want to treat it like observations --- the objective is not to set up a simulation that is somehow "consistent" with the coupled system that JRA55 was originally run with.

Reply:
It would be interesting to see the difference between the two formulations (maybe it is not that large). These "similiarity functions" are all parameterizations that include empirical elements or relations that are extrapolated from data, so rigorous mathematical consistency does not necessarily equate to results that are closer to observations.

Reply:
That's not a bad idea, in that case something like 

```julia
inline profile(ψ::SimplifiedProfile, h, ℓ, L) = ...
```

might work.

PS this is the profile, not the bulk coeffcient right? It's what we get when we integrate the similarity gradient.

> rigorous mathematical consistency does not necessarily equate to results that are closer to observations.

The purpose of including the term is not to be "rigorous". It's to avoid "random" approximations that rely on hidden assumptions. I think the same argument is actually the real reason to include it. If including the extra term means that this expression generalizes to cases with larger ``ℓ`` or small ``L`` --- but does not affect the results for typical values of these parameters --- that is exactly why we should include it. Approximations need to have a purpose. The purpose can't be to drop a term in a mathematical expression. It would actually have to make a difference in computational performance or somethign like that to justify this.